packages/edit-site/src/components/app/index.js

```diff
@@ -50,14 +44,9 @@ export default function App() {
 	}
 
 	return (
-		<SlotFillProvider>
```

Comment:
Yes, If I'm not wrong it's not really needed at the top level because of a refactor that I did to Slot/Fill at some point where slots will just get inserted under body anyway.

Reply:
I couldn't find where we add it. Do you mean [here](https://github.com/WordPress/gutenberg/pull/56779)?

In edit-post we still have this top level.

Reply:
> In edit-post we still have this top level.

Maybe it's not needed there as well. What issues does it create to drop it?

Reply:
I didn't observe any, but wanted to understand it better.

Reply:
Could this removal be the cause to this issue? https://github.com/WordPress/gutenberg/issues/67538

Reply:
Actually I restored it before merging but I'll take a look at that regression tomorrow.

Reply:
Confrming the issue: https://github.com/WordPress/gutenberg/issues/67538#issuecomment-2515465773